controllers/review/linkAction/ReviewNotesLinkAction.inc.php

```diff
@@ -24,9 +24,9 @@ class ReviewNotesLinkAction extends LinkAction {
 	 * to show information about.
 	 * @param $submission Submission The reviewed submission.
 	 * @param $user User The user.
-	 * @param $icon String the icon to use
+	 * @param status String Review's read status
```

Comment:
`$status` looks to take values `new` or [anything else]. Mind adding that to the comment? Or better yet, if it's basically a boolean flag, actually using a boolean? Better to avoid magic strings if possible, IMO.